core/citrus-base/src/test/java/com/consol/citrus/functions/FunctionsTest.java

```diff
@@ -102,4 +96,9 @@ public void testRandomStringWithParams() throws Exception {
         Assert.assertEquals(randomString(10L, RandomStringFunction.LOWERCASE, context).length(), 10);
         Assert.assertEquals(randomString(10L, RandomStringFunction.UPPERCASE, false, context).length(), 10);
     }
+
+    @Test
+    public void testUnixTimestamp() throws Exception {
+        Assert.assertEquals(String.valueOf(System.currentTimeMillis() / 1000L), unixTimestamp(context));
```

Comment:
from a second look, what was your thought of using `Instant`s [here](https://github.com/citrusframework/citrus/pull/870/files#diff-da30f2b2fdfea86a909e000542c32cf4a64360004c873da28c8fbcc7530fe872R31)? why aren't you using `System.currentTimeMillis()` as well?